client/my-sites/controller.js

```diff
@@ -348,13 +347,6 @@ function onSelectedSiteAvailable( context ) {
 			return false;
 		}
 		context.hideLeftNavigation = true;
-	} else {
```

Comment:
@andres-blanco - I think we want to keep this loading state in place. We do use it for Migrate Guru migrations, mostly because the site could be in really weird states before the migration completes.

Reply:
Yeah, I was hesitant about that. 
Where do you think we should point users to? `/migrate` uses Jetpack migration and pointing users to `/setup/hosted-site-migration` doesn't make sense for ongoing migrations.

Reply:
I think that UI shows an overlay when a migration is in progress -- see https://github.com/Automattic/wp-calypso/pull/88973 for some previous changes to that UX.